card/spec/support/helper/render_helper.rb

```diff
@@ -14,10 +14,10 @@ def render_content content, format_args={}
         render_content_with_args content, format_args
       end
 
-      def render_content_with_args content, format_args={}, view_args={}
+      def render_content_with_args content, format_args={}
         @card ||= Card.new name: "Tempo Rary 2"
         @card.content = content
-        @card.format(format_args)._render :core, view_args
+        @card.format(format_args)._render :core
```

Comment:
I don't see a reason to keep this method.

Reply:
good point.  removing.